homeassistant/components/netgear_lte/sensor_types.py

```diff
@@ -3,6 +3,11 @@
 SENSOR_SMS = 'sms'
 SENSOR_USAGE = 'usage'
 
-ALL = [SENSOR_SMS, SENSOR_USAGE]
+SENSOR_UNITS = {
+    SENSOR_SMS: 'unread',
+    SENSOR_USAGE: 'MiB',
+}
+
+ALL = SENSOR_UNITS.keys()
```

Comment:
```suggestion
ALL = list(SENSOR_UNITS)
```

Reply:
Okay. I actually think `keys()` is clearer in this case, why do you prefer `list()`?

Reply:
It doesn't always matter, but the two ways don't return the same thing.
`list(d)` will return a list of the dictionary keys. `d.keys()` returns a `dictkeys` instance. The latter doesn't support all things a list supports, eg indexing.
The third way, which is supposed to be faster in some cases, and is supported from Python3.5 is unpacking, `[*d]`. This will also return a list.

Reply:
`list(d)` is also one character shorter to write than `d.keys()`. :smile: 😎

Reply:
LOL ... but the result of `.keys()` probably takes less memory :)

Thanks, those are good points that I did not really consider.